src/client/containers/LandingPage/LandingPage.container.js

```diff
@@ -1,15 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './LandingPage.styles.css';
-import { useProducts } from '../../hooks/useProducts';
-import Products from '../../components/Products/Products.component';
+import { useProducts, PRODUCT_PER_PAGE } from '../../hooks/useProducts';
+import Loader from '../../components/Loader/Loader.component';
+import Sort from '../../components/Sort/Sort.component';
+import Pagination from '../../components/Pagination/Pagination.component';
+import SpecialOfferBanner from '../../components/SpecialOfferBanner/SpecialOfferBanner.component';
+import ProductsView from '../../components/ProductsView/ProductsView.component';
 
 export default function LandingPage() {
-  const { products, isLoading, error } = useProducts();
+  const [sortingPreferences, setSortingPreferences] = useState({
+    sortKey: 'name',
+    sortOrder: 'asc',
+  });
+  const { isLoading, error, products, totalCount, loadMoreProducts } =
+    useProducts(sortingPreferences);
```

Comment:
Here we should pass in `currentPage` to let the data hook load, sort and paginate for us anytime the sort order or the page changes.